app/controllers/miq_ae_class_controller.rb

```diff
@@ -2554,14 +2554,12 @@ def domain_overrides
   end
 
   def get_session_data
-    @layout     = "miq_ae_class"
     @title      = _("Datastore")
     @lastaction = session[:aeclass_lastaction]
     @edit       = session[:edit]
   end
 
   def set_session_data
-    session[:aeclass_lastaction] = @lastaction
     session[:edit]               = @edit
```

Comment:
You can remove this line because it done for every controller here https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller.rb#L1941 .